report/assets/styles.css

```
@@ -703,14 +704,27 @@
 .lh-metric__description {
   display: none;
   grid-column-start: 2;
-  grid-column-end: 3;
+  grid-column-end: 4;
   color: var(--report-text-color-secondary);
 }
 
 .lh-metric__value {
+  font-size: var(--metric-value-font-size);
+  margin: 5px 0;
```

```suggestion
  margin: calc(var(--default-padding) / 2) 0;
```

seemed like it could use an extra pixel and always love to avoid arbitrary padding when possible :)